interactive.py

```diff
@@ -111,18 +105,24 @@ def curses_main(stdscr):
 
     while True:
 
-        # todo: none of this resizing works
-        (n_rows, n_cols) = map(int, stdscr.getmaxyx())
+        # TODO: handle resizing.  Need to (1) figure out how to reliably get
+        # the terminal size -- the recommended method doesn't seem to work
+        #    (n_rows, n_cols) = map(int, stdscr.getmaxyx()) ...doesn't work
+        #    ...map(int, os.popen('stty size', 'r').read().split() ...may work
+        # and then (2) implement the logic to resize all the subwindows as above
+
         stdscr.clear()
         linecap = n_cols - 1
         logscreen_height = n_rows - (header_height + jobs_height + dirs_height)
 
         elapsed = (datetime.datetime.now() - last_refresh).total_seconds() 
+        full_refresh = False
         if (elapsed < refresh_period):
```

Comment:
This one is a little awkward because it's not recording that the refresh is needed, in order to do the refresh later, it's recording that we're doing the refresh right here and now, in order to display that fact later.  So `doing_full_refresh` or `full_refresh_done` might be more accurate.  Thoughts?

Reply:
Makes sense.